config/database.yml

```diff
@@ -11,14 +11,12 @@ default: &default
 
 development:
   <<: *default
-  database: education_development
```

Comment:
That file uses env vars. The dotenv gem was in the gemfile but wasn't being used. Travis was reading those vars from `.travis.yml` and it was causing builds to fail. Those vars should be set during staging/production deploys, right?